utils/ips.go

```diff
@@ -122,6 +127,18 @@ type inetInterfaceSpec struct {
 	IPv6 bool
 }
 
+// -- matches static
+type staticInterfaceSpec struct {
+	Spec string
+	Name string
+	IP net.IP
+}
+
+func (s staticInterfaceSpec) Match(index int, iip interfaceIP) bool {
+	// Never matches
```

Comment:
What is the reason why this never matches?

It looks like the logic for this spec is in the inetInterfaceSpec instead.